composer.json

```diff
@@ -16,7 +16,7 @@
 	},
 
 	"require": {
-		"php": "^5.3|^7.0"
+		"php": "^5.4|^7.0"
```

Comment:
It's already reverted.
Can you take a look at the other question if you have the time ?